src/main/k/in-progress/tests/list-unify-1/tests/2.output.golden

```diff
@@ -0,0 +1,15 @@
+<generatedTop>
```

Comment:
I think the output here would include b, as (unlike for set) you're only removing from one end.

Reply:
you are right, I got confused between lists and sets. fixed now.